File: A1/A1-012/A1-012.py
# --------------------------------------------------
# File Name : A1-012.py
# Problem   : Reverse a number
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input the number and the operation
num = int(input().strip())
operation = input().strip()

# Reverse a number
reversed_num = int(str(num)[::-1])

# Calculate result
result = 0
if operation == "+":
    result = num + reversed_num
elif operation == "*":
    result = num * reversed_num

# Output
print(f"{num} {operation} {reversed_num} = {result}")

File: A1/A1-012/A1-012.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int num01;
    char operation;
    cin >> num01 >> operation;
    // Reverse 2-digit number
    int num02 = ((num01 % 10) * 10) + (num01 / 10);
    // Output
    if(operation == '+')
    {
        cout << num01 << " + " << num02 << " = " << num01 + num02;
    }
    else if(operation == '*')
    {
        cout << num01 << " * " << num02 << " = " << num01 * num02;
    }
    return 0;
}